shared/wallets/confirm-form/participants/index.js

```diff
@@ -76,7 +76,7 @@ const Participants = (props: ParticipantsProps) => {
     <Kb.Box2 direction="vertical" fullWidth={true}>
       <ParticipantsRow heading="From">
         <AccountEntry
-          contents={props.fromAccountAssets}
+          contents={props.fromAccountAssets + ' available'}
```

Comment:
hm, this seems like a string that needs to come from core. since the '+ more' is coming from there too, here's what it looks like for me:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/11968340/50114149-d69fb080-0211-11e9-9900-e7fa829b2b37.png">

Reply:
and because of this line in the ticket
> If I were to send another asset, say BTC/Stronghold.com, it would say "35 BTC/Stronghold.com available".

Reply:
ok thanks